### ISSUE TITLE
Updating inventory quantity for given marketplaces

### ISSUE THREAD

**Issue by FinishingLine** [jlevers/selling-partner-api#644]
## Problem description:
Not sure if I'm missing anything, but when doing the below:
````
$data = [
    'product_type' => 'PRODUCT',
    'patches' => [
        new PatchOperation([
            'op' => 'replace',
            'path' => '/attributes/fulfillment_availability',
            'value' => [
                [
                    'quantity' => $currentStockLevel,
                    'fulfillment_channel_code' => 'DEFAULT',
                ],
            ],
        ]),
    ],
];
````

All marketplaces at Amazon receive the same inventory level - my question is, is it possible to limit which marketplaces are updated with the stock levels? Mainly as some listings on international marketplaces are turning back on after we update the inventory (but would prefer not to have the stock on)

**Comment by misterakko** [comment]
Yes, it's possibile and I found it to be hard. Unfortunately the company I did that particular development for — because they had separate warehouses in different EU member countries — went bankrupt during the COVID pandemic (they sold tires, and nobody was driving around), so I only have vague memories.
But basically, you must create separate SKUs for the separate marketplaces, related to the same ASINs, and send updates for SKUs and not for ASINs. At that point you can tell Amazon SPAPI that SKU x is in stock with ten items available in Italy, zero in Germany, while SKU y is availabile in Germany but not in Italy.

**Comment by iajrz** [comment]
Latest major version 7.x was released in June 19th 2024.
Inactive issues from before that time are getting closed.
If somehow this is still relevant, please reopen the issue.
